bvm/ballerina-config/src/main/java/org/ballerinalang/config/ConfigProcessor.java

```diff
@@ -178,4 +183,14 @@ private static ParseTree buildParseTree(CharStream configEntriesStream) {
         TomlParser parser = new TomlParser(tokenStream);
         return parser.toml();
     }
+
+    private static void addErrorMsg(StringBuilder errMsgBuilder, String msg, String key, String val) {
+        errMsgBuilder.append("ballerina: ");
+        errMsgBuilder.append(msg);
+        errMsgBuilder.append(" '");
+        errMsgBuilder.append(key);
+        errMsgBuilder.append("': ");
+        errMsgBuilder.append(val);
+        errMsgBuilder.append('\n');
```

Comment:
Can we use the StringBuilder chain appending here? `errMsgBuilder.append("ballerina: ").append(msg).append(" '").append(key).append("': ").append(val).append('\n');`